### ISSUE TITLE
Ray tracing support

### ISSUE THREAD

**Issue by ldoppertin** [Unity-Technologies/com.unity.demoteam.hair#102]
I've seen there is an older ray tracing support branch with an open pull request.
Are there any plans to support ray tracing in the near future?

**Comment by fuglsang** [comment]
Yes, though not immediate near future. I'm currently working through existing features that need a bit more attention/tweaking before a 1.0 release. Ray tracing will be merged in (branch is quite outdated so will need to comb a bit) after some much requested basic documentation, most likely.
